hbci_client.py

```diff
@@ -21,6 +26,42 @@
 version = subprocess.check_output(["git", "describe", "--abbrev=5", "--always"]).decode('utf8').strip()
 version = version[:5]
 
+
+def encoded_png_to_unicode_qr(encoded_png, scaling_factor=1.0):
```

Comment:
love bits like this

Care to add a test function with one hardcoded input & CLI output? 🥺